ocs_ci/ocs/ui/page_objects/virtualmachine_ui.py

```
@@ -0,0 +1,405 @@
+        """
+        logger.info(f"Verifying namespace '{namespace}' is gone from left-side tree")
+        ns_xpath = self.vm_locators["namespace_left_tree_item_tmpl"].format(
+            namespace=namespace
+        )
+        end = time.time() + timeout
+        while time.time() < end:
+            try:
+                els = self.driver.find_elements(By.XPATH, ns_xpath)
+                if not els or not any(e.is_displayed() for e in els):
+                    logger.info(
+                        f"Namespace '{namespace}' no longer visible in left tree"
```

Ruff (0.15.21)

[warning] Logging statement uses f-string

(G004)